lib/hotwire/spark/file_watcher.rb

```diff
@@ -25,7 +25,13 @@ def expand_path(path)
     end
 
     def paths
-      @callbacks_by_path.keys
+      ensure_paths_present @callbacks_by_path.keys
+    end
+
+    def ensure_paths_present(paths)
```

Comment:
Would you mind calling this `only_existing_paths`. This is an in-house convention, but we usually use `ensure` to name methods that may raise or take other action, not those that purely return values.

Reply:
Yup. Makes sense! That's a better name.